2023-04-07 09:24:02,857 INFO    MainThread:339 [internal.py:wandb_internal():93] W&B internal server running at pid: 339, started at: 2023-04-07 09:24:02.857142
2023-04-07 09:24:02,859 INFO    WriterThread:339 [datastore.py:open_for_write():75] open: /kaggle/working/adacos_20230407/wandb/run-20230407_092401-31oeilhz/run-31oeilhz.wandb
2023-04-07 09:24:02,861 DEBUG   SenderThread:339 [sender.py:send():236] send: header
2023-04-07 09:24:02,861 DEBUG   SenderThread:339 [sender.py:send():236] send: run
2023-04-07 09:24:03,217 INFO    SenderThread:339 [dir_watcher.py:__init__():216] watching files in: /kaggle/working/adacos_20230407/wandb/run-20230407_092401-31oeilhz/files
2023-04-07 09:24:03,217 INFO    SenderThread:339 [sender.py:_start_run_threads():818] run started: 31oeilhz with start time 1680859441
2023-04-07 09:24:03,219 DEBUG   SenderThread:339 [sender.py:send():236] send: summary
2023-04-07 09:24:03,220 DEBUG   HandlerThread:339 [handler.py:handle_request():140] handle_request: check_version
2023-04-07 09:24:03,220 INFO    SenderThread:339 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-04-07 09:24:03,221 DEBUG   SenderThread:339 [sender.py:send_request():250] send_request: check_version
2023-04-07 09:24:03,360 DEBUG   HandlerThread:339 [handler.py:handle_request():140] handle_request: run_start
2023-04-07 09:24:03,370 DEBUG   HandlerThread:339 [meta.py:__init__():35] meta init
2023-04-07 09:24:03,370 DEBUG   HandlerThread:339 [meta.py:__init__():49] meta init done
2023-04-07 09:24:03,370 DEBUG   HandlerThread:339 [meta.py:probe():209] probe
2023-04-07 09:24:03,380 DEBUG   HandlerThread:339 [git.py:repo():29] git repository is invalid
2023-04-07 09:24:03,380 DEBUG   HandlerThread:339 [meta.py:_save_code():87] save code
2023-04-07 09:24:03,380 WARNING HandlerThread:339 [meta.py:_save_code():89] unable to save code -- program entry not found
2023-04-07 09:24:03,380 DEBUG   HandlerThread:339 [meta.py:_save_pip():53] save pip
2023-04-07 09:24:03,381 DEBUG   HandlerThread:339 [meta.py:_save_pip():67] save pip done
2023-04-07 09:24:03,381 DEBUG   HandlerThread:339 [meta.py:_save_conda():74] save conda
2023-04-07 09:24:04,220 INFO    Thread-7  :339 [dir_watcher.py:_on_file_created():269] file/dir created: /kaggle/working/adacos_20230407/wandb/run-20230407_092401-31oeilhz/files/wandb-summary.json
2023-04-07 09:24:04,220 INFO    Thread-7  :339 [dir_watcher.py:_on_file_created():269] file/dir created: /kaggle/working/adacos_20230407/wandb/run-20230407_092401-31oeilhz/files/requirements.txt
2023-04-07 09:24:04,221 INFO    Thread-7  :339 [dir_watcher.py:_on_file_created():269] file/dir created: /kaggle/working/adacos_20230407/wandb/run-20230407_092401-31oeilhz/files/conda-environment.yaml
2023-04-07 09:24:09,219 INFO    Thread-7  :339 [dir_watcher.py:_on_file_created():269] file/dir created: /kaggle/working/adacos_20230407/wandb/run-20230407_092401-31oeilhz/files/output.log
2023-04-07 09:24:13,221 INFO    Thread-7  :339 [dir_watcher.py:_on_file_modified():286] file/dir modified: /kaggle/working/adacos_20230407/wandb/run-20230407_092401-31oeilhz/files/output.log
2023-04-07 09:24:17,223 INFO    Thread-7  :339 [dir_watcher.py:_on_file_modified():286] file/dir modified: /kaggle/working/adacos_20230407/wandb/run-20230407_092401-31oeilhz/files/output.log
2023-04-07 09:24:33,544 WARNING MainThread:339 [internal.py:wandb_internal():153] Internal process interrupt: 1
